src/common/regions/utils.ts

```diff
@@ -192,10 +183,17 @@ export function getGeolocatedRegions(
   if (geolocation.country !== UNITED_STATES) {
     return null;
   } else {
+    const county = getCountyRegionFromZipCode(
+      geolocation.zipCode,
+      countyToZipMap,
```

Comment:
this is going to help if we try getting zip code mappings by state or county on demand